api/v1beta1/nodemaintenance_types.go

```diff
@@ -46,8 +46,10 @@ type NodeMaintenanceSpec struct {
 	// Important: Run "make" to regenerate code after modifying this file
 
 	// Node name to apply maintanance on/off
+	//+operator-sdk:csv:customresourcedefinitions:type=spec
 	NodeName string `json:"nodeName"`
 	// Reason for maintanance
+	//+operator-sdk:csv:customresourcedefinitions:type=spec
```

Comment:
Interesting. Where is this documented?  Odd that each field needs the same annotation

Reply:
See [here](https://sdk.operatorframework.io/docs/building-operators/golang/references/markers/).
(I have also updated the link in the PR's summary)